packages/ui/src/components/SelectableCard/index.tsx

```diff
@@ -205,6 +285,8 @@ export const SelectableCard = forwardRef(
           data-testid={dataTestId}
           data-type={type}
           data-has-label={!!label}
+          data-has-illustration={!!illustration}
+          data-has-icon={!!productIcon && !illustration}
```

Comment:
you could merge those in one data, like `data-image="illustration | product"`

Reply:
styling is a little bit different for illustrations and icons

Reply:
yes exactly so you could do like you did but merging both, something like:

```css
  &[data-image="illustration"] {
    padding: ${({ theme }) => theme.space[0]};
  }
  &[data-image="product"] {
    padding: ${({ theme }) => theme.space[0]};
    padding-right: ${({ theme }) => theme.space['2']};
  }
```